### PR TITLE
Fixed UI tests

### PR DESCRIPTION
**Pull request by bonkey**
- Added automatic login to UI tests
- Added logout on demo app start
- Added conditional response from /customer based on “Authorized:” header: accepts any token
- Added AtlasMockAPIRunner – dumb-simple macOS command-line app to test AtlasMockAPI manually

### CODE REVIEW

File: AtlasMockAPI/AtlasMockAPI/JSON/GET*|config.json [hunk 1]
```diff
@@ -6,6 +6,6 @@
         "url": "http://localhost:9080"
     },
     "oauth2-provider": {
-        "url": "http://localhost:9080/oauth2/authorize?realm=/customers&response_type=token&client_id=CLIENT_ID&redirect_uri=http://com.atlas.app/redirect"
+        "url": "https://planb-provider-test.greendale-questionmark.zalan.do/oauth2/authorize?realm=/customers&response_type=token&client_id=atlas_Y2M1MzA&redirect_uri=http://de.zalando.atlas.AtlasCheckoutDemo/redirect"
     }
```
Comment (ahmed-shehata) [inline AtlasMockAPI/AtlasMockAPI/JSON/GET*|config.json]: Why point to the staging here?

Comment (bonkey) [inline AtlasMockAPI/AtlasMockAPI/JSON/GET*|config.json]: cuz mocked login is not ready yet.

Feel free to implement POST in the server. GET is _probably_ there. Probably, as I didn't even run it, but prepared something for a start.

Just couldn't think about having broken tests during weekend ;-)

**Comment by bonkey** [comment]
👍

**Comment by ahmed-shehata** [comment]
👍

**Pull request merged**